test/e2e/utils/handler.go

```diff
@@ -83,10 +83,6 @@ const (
 
 func Handle(testParams *TestParameters) error {
 	// Validating the test parameters.
-	// ZB uses gRPC as the client protocol, so if ZB is enabled, the client protocol must be gRPC.
```

Comment:
not sure i complete understand this check removal. ZB only works with grpc correct?

Reply:
sync offline, ZB only supports grpc. but we are enabling all the test cases exercised via the http option. under the hood, gcsfuse automatically uses grpc client for zb bucket.